### DISCUSSION TITLE
metatube-server 报404

### DISCUSSION THREAD

**Discussion by iamckun** [metatube-community/jellyfin-plugin-metatube#129] [Ideas]
![image](https://user-images.githubusercontent.com/65990499/218038318-6b16a9f2-9b00-4b9b-b7f7-3ff3ade958e6.png)

**Comment by iamckun** [comment]
解决了，希望可以加一个配置代理得地方

**Reply by ootank2018** [reply]
能说下怎么解决的吗？

**Reply by iamckun** [reply]
挂了一个代理，docker得http_proxy 没代理成功我建了一个macvlan用得旁路游

**Reply by zhayo0** [reply]
我是全局代理的。要单独设置代理吗？

**Comment by zhayo0** [comment]
我是全局代理的。能刮削到一部分。很多也报404.用的免费后端。也是报404

**Reply by iamckun** [reply]
你去容器里curl看看能不能通那些网站，能得话后端就是好的，另外一个地方我回复你了可以看看是不是那个问题

**Reply by zhayo0** [reply]
冒失fanza网站打不开.导致刮削失败了
